src/install.rs

```diff
@@ -233,6 +236,53 @@ fn write_platform(mountpoint: &Path, platform: &str) -> Result<()> {
     Ok(())
 }
 
+/// Copy networking config if asked to do so
+fn copy_network_config(mountpoint: &Path, net_config_src: &str) -> Result<()> {
+    eprintln!("Copying networking configuration from {}", net_config_src);
+
+    // get the path to the destination directory
+    let mut net_config_dest = mountpoint.to_path_buf();
+    net_config_dest.push("coreos-installer-network");
+
+    // make the directory if it doesn't exist
+    create_dir_all(&net_config_dest)
+        .chain_err(|| {
+            format!(
+                "creating destination networking config directory {}",
+                net_config_dest.display()
+            )
+        })?;
+
+    // if the directory has files in it then clear them out
+    for entry in read_dir(&net_config_dest)
+        .chain_err(|| format!("reading directory {}", net_config_dest.display()))?
+    {
+        let entry = entry
+            .chain_err(|| format!("reading directory {}", net_config_dest.display()))?;
+        if entry.path().is_file() {
+            eprintln!("Removing file {}", entry.path().display());
+            remove_file(entry.path()).
+                chain_err(|| format!("removing {}", entry.path().display()))?;
+        }
+    }
+    
+    // copy files from source to destination directories
+    for entry in read_dir(&net_config_src)
+        .chain_err(|| format!("reading directory {}", net_config_src))?
+    {
+        let entry = entry
+            .chain_err(|| format!("reading directory {}", net_config_src))?;
+        let srcpath = entry.path();
+        let destpath = net_config_dest.join(entry.file_name());
+        if srcpath.is_file() {
+            eprintln!("Copying {} to installed system", srcpath.display());
```

Comment:
This seems too verbose.  We're already announcing that we're copying the parent directory.

Reply:
I think it does more good than harm because a user could be claiming that they set up networking and it wasn't getting propagated properly. Well, how did they set up networking? Did they use nmtui? Did the copy files around themselves? Did they use `ip` directly?

This little bit of information let's us know at least what file(s) got copied over, if any.

Reply:
In the vast majority of cases there will probably only be 1 interface so one extra line of logging output.

Reply:
Okay, makes sense.